db/migrate/20240517145354_create_sign_in_certificates.rb

```diff
@@ -0,0 +1,15 @@
+class CreateSignInCertificates < ActiveRecord::Migration[7.1]
+  def change
+    create_table :sign_in_certificates do |t|
+      t.string :issuer, null: false
+      t.string :subject, null: false
+      t.string :serial, null: false
+      t.datetime :not_before, null: false
+      t.datetime :not_after, null: false
+      t.text :plaintext, null: false
+      t.references :config, polymorphic: true, null: false
```

Comment:
Hmmm yeah I think we should remove the polymorphic component and just have two associations, one for each of the configs. I don't like that the possible associations aren't embedded in the schema itself (the schema doesn't define what tables these references are associated to)

Reply:
I don't feel strongly about the polymorphic relationship. I think it makes sense in some cases, but maybe not in this one. It's unlikely we'll have configuration types other than client or service account for some time; if we do, we'll just need to add a reference column on this table to the new configuration table.